apps/trading-e2e/src/support/step_definitions/vega-wallet.step.js

```diff
@@ -28,3 +28,16 @@ When('I connect to Vega Wallet', () => {
   );
   vegaWallet.clickConnectVegaWallet();
 });
+
+When('I open wallet dialog', () => {
+  vegaWallet.validatePublicKeyDisplayed('47836c…c7d278'); // Default Test wallet pub key
```

Comment:
chore (blocking): can we move both these keys to the env file so developers can configure their own local wallet when running tests

Reply:
I've added the second pub key into a Cypress env and now truncating it in the method. 👍🏼